src/extension.ts

```diff
@@ -379,7 +379,8 @@ class SquashExtention {
     }
 
     async waitForDebugConfigWithImage(image: string, dbgr: string): Promise<string> {
-        const result = await squash<squashinterface.DebugRequest>(`debug-request ${image} ${dbgr}`);
+        let pname = get_conf_or("process-name", "")
+        const result = await squash<squashinterface.DebugRequest>(`debug-request ${image} ${dbgr} ${pname}`);
```

Comment:
this should be -processname, right?

Reply:
Right now full config option name will be vs-squash.process-name and it seems to comply with others.
In general, we have 2 naming conventions there:
vs-squash.squash-path
vs-squash.remotePath